Description: Turn recode conflicts error message into a warning. The
 recode extension is packaged as a shared library in Debian.
Origin: vendor
Forwarded: no
Last-Update: 2010-01-18

--- a/ext/recode/config9.m4
+++ b/ext/recode/config9.m4
@@ -13,6 +13,6 @@ if test "$PHP_RECODE" != "no"; then
   fi
 
   if test -n "$recode_conflict"; then
-    AC_MSG_ERROR([recode extension can not be configured together with:$recode_conflict])
+    AC_MSG_WARN([recode extension can not be used together with:$recode_conflict])
   fi
 fi
